src/app/pages/password-reset/password-reset.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { PasswordResetService } from 'src/app/services/password/password-reset.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ToastrService } from 'ngx-toastr';
+
+@Component({
+  selector: 'app-password-reset',
+  templateUrl: './password-reset.component.html',
+  styleUrls: ['./password-reset.component.scss']
+})
+export class PasswordResetComponent implements OnInit {
+  resetForm: FormGroup;
+  email: string;
+  success: boolean;
+  disabled: boolean = true;
+
+  constructor(
+    private resetService: PasswordResetService,
+    private fb: FormBuilder,
+    private toastrService: ToastrService,
+    ) {}
+
+  ngOnInit() {
+    this.resetForm = this.fb.group({
+      email: ['', [Validators.required, Validators.email]]
+    });
+    this.resetForm.valueChanges.subscribe( value => {
+      this.disabled = this.resetForm.invalid;
+    });
+  }
+
+  onSubmit() {
+    this.email = this.resetForm.get('email').value;
```

Comment:
Good work Joel, could you please provide progress updates to the user on what's happening when they submit an email, I realize on slow 3G the process takes really long and a user can easily leave the page thinking something went wrong, You could for example change the button text to `please wait`, then disable it at least until the response comes back.